### PR TITLE
Replace Cilium LoadBalancer with MetalLB

### PR DESCRIPTION
**Pull request by bschimke95**
This PR replaces the Cilium LoadBalancer with MetalLB in the default Canonical Kubernetes configuration. The Cilium LoadBalancer had a bunch of shortcomings, e.g. IPv6 support.

### CODE REVIEW

**Review by berkayoz — APPROVED** [review]
LGTM, we can merge after verifying we can reach ingress or gateway services through a loadbalancer ip assigned by metallb.

**Review by louiseschmidtgen — COMMENTED** [review]
Nice work Ben! little suggestion

File: tests/integration/tests/test_util/util.py
Comment (bschimke95) [reply tests/integration/tests/test_util/util.py:445]: the `excluded_ips` are single IPs. A CIDR contains a range of IPs.
Your suggestion implies that the excluded parameter contains CIDRs, right?

File: .github/workflows/integration.yaml
Comment (eaudetcobello) [inline .github/workflows/integration.yaml:85]: Revert before merge

File: tests/integration/tests/test_util/util.py
Comment (eaudetcobello) [inline tests/integration/tests/test_util/util.py:216]: Can we make this less than 20m? I don't think it looks too good on us to have long waits like this. (Just to be clear, this is not your fault).

File: .github/workflows/integration.yaml
Comment (HomayoonAlimohammadi) [inline .github/workflows/integration.yaml:106]: Just a reminder to delete before merging.

**Pull request merged**
